README.md

```diff
@@ -100,38 +115,70 @@ Also, make sure your firewall is configured to allow inbound connections to port
 
 ### Binaries (Windows/Linux/macOS)
 
-Binary releases are provided for common operating systems and architectures:
+Binary releases are provided for common operating systems and architectures.
+The easiest method is to download Decrediton from the link below, which will include dcrd.
+Advanced users may prefer the Command-line app suite, which includes dcrd and dcrwallet.
 
 https://decred.org/downloads
 
+* How to verify binaries before installing: https://docs.decred.org/advanced/verifying-binaries/
+* How to install the CLI Suite: https://docs.decred.org/wallets/cli/cli-installation/
+* How to install Decrediton: https://docs.decred.org/wallets/decrediton/decrediton-setup/
+
 ### Build from source (all platforms)
 
-Building or updating from source requires the following build dependencies:
+<details><summary><b>Install Dependencies</b></summary>
```

Comment:
This is a markdown file.  It shouldn't have HTML in it.

Reply:
Understood. I can remove it.
What's the purpose of line 112: "<a name="Installation" />"?